### PR TITLE
Fix pint merge conflicts from 3.2.1-post

### PR DESCRIPTION
**Pull request by joel-hamill**

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -57,27 +237,27 @@ Start Zookeeper. You'll need to keep this service running throughout, so use a d
         -e ZOOKEEPER_CLIENT_PORT=32181 \
         confluentinc/cp-zookeeper:3.3.0
 
-  This command instructs Docker to launch an instance of the ``confluentinc/cp-zookeeper:3.3.0`` container and name it ``zookeeper``.  We also specify that we want to use host networking and pass in the required parameter for running Zookeeper: ``ZOOKEEPER_CLIENT_PORT``.  For a full list of the available configuration options and more details on passing environment variables into Docker containers, see the `configuration reference docs <configuration.html>`_.
+  This command instructs Docker to launch an instance of the ``confluentinc/cp-zookeeper:3.2.1`` container and name it ``zookeeper``.  You also specify that you want to use host networking and pass in the required parameter for running ZooKeeper: ``ZOOKEEPER_CLIENT_PORT``.  For a full list of the available configuration options and more details on passing environment variables into Docker containers, see the `configuration reference docs <configuration.html>`_.
```
Comment (aayars) [inline docs/quickstart.rst]: Looks like version is getting stomped on (3.2.1 vs 3.3.0)

File: docs/quickstart.rst [hunk 2]
```diff
@@ -3,30 +3,210 @@
...
+
+  .. Note::
+
+    To get started, you'll need to first `install Docker and get it running <https://docs.docker.com/engine/installation/>`_.  The CP Docker Images require Docker version 1.11 or greater.
+
+If you're running on Windows or macOS, you'll need to use `Docker Machine <https://docs.docker.com/machine/install-machine/>`_ to start the Docker host.  Docker runs natively on Linux, so the Docker host will be your local machine if you go that route.  If you are running on Mac or Windows, be sure to allocate at least 4 GB of ram to the Docker Machine.
```
Comment (aayars) [inline docs/quickstart.rst]: Just "Mac" here (not macOS)

File: docs/quickstart.rst [hunk 3]
```diff
@@ -3,30 +3,210 @@
...
+.. _quickstart_compose:
+
+Getting Started with Docker Compose
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Docker Compose is a powerful tool that enables you to launch multiple Docker images in a coordinated fashion.  It is ideal for platforms like Confluent.  Before you get started, you will need to install both the core `Docker Engine <https://docs.docker.com/engine/installation/>`_ and `Docker Compose <https://docs.docker.com/compose/install/>`_.  Once you've done that, you can follow the steps below to start up the Confluent Platform services.
+
+#. Create and configure the Docker Machine.
+
+   **Important:** If you are using Docker for Mac or Docker for Windows, you can skip this step.
```
Comment (aayars) [inline docs/quickstart.rst]: You may have this backwards -- Linux users don't need docker-machine. I'm not sure why we're recommending it for Mac users either, TBH -- docker-machine is an extra piece that needlessly complicates the Docker experience.

File: docs/quickstart.rst (hunk 2 above)
Comment (joel-hamill) [reply docs/quickstart.rst]: @aayars aren't we talking specifically about the Apple OS? I believe macOS is correct https://en.wikipedia.org/wiki/Macintosh_operating_systems

File: docs/quickstart.rst (hunk 3 above)
Comment (joel-hamill) [reply docs/quickstart.rst]: Docker for Mac and Windows take care of this as well. So can the step be removed entirely?

File: docs/quickstart.rst (hunk 2 above)
Comment (aayars) [reply docs/quickstart.rst]: Ah. I thought it typically referred to "old" Mac OS, pre-OS X. Guess it's making a comeback.

Who am I to argue with Wikipedia.

File: docs/quickstart.rst (hunk 3 above)
Comment (aayars) [reply docs/quickstart.rst]: If we're sure the steps in the quickstart work without it, I think so. We should be sure there aren't other changes needed.

File: docs/quickstart.rst (hunk 2 above)
Comment (joel-hamill) [reply docs/quickstart.rst]: lol...that all-knowing authority

File: docs/quickstart.rst (hunk 3 above)
Comment (joel-hamill) [reply docs/quickstart.rst]: i ran through this extensively on mac

**Pull request merged**